src/Pages/Cart/Cart.js

```diff
@@ -0,0 +1,317 @@
+
+  updateCart = res => {
+    if (res.message === 'EMPTY') {
+      this.setState({
+        carts: [],
+      });
+    } else if (res.message === 'SUCCESS') {
+      this.setState({
+        orderNumber: res.data.orderNumber,
+        carts: res.data.carts,
+      });
+    }
+  };
+
+  //이거 지우나?
+  // getSubTotal = () => {
+  //   this.setState({ subtotal: 0 });
+  //   let priceArr = [];
+  //   this.state.carts.map(item => {
+  //     priceArr.push(item.productPrice - 0);
+  //   });
+
+  //   const subtotal = priceArr.reduce((a, b) => a + b);
+
+  //   this.setState({ subtotal });
+  // };
+
+  testTOTAL = () => {
+    let priceArr = [];
+
+    for (let i = 0; i < this.state.carts.length; i++) {
+      priceArr.push(
+        Number(this.state.carts[i].productPrice) *
+          this.state.carts[i].productQuantity,
+      );
+    }
+
+    const subtotal = priceArr.reduce((a, b) => a + b);
+
+    this.state.subtotal !== subtotal && this.setState({ subtotal });
+  };
+
+  componentDidUpdate() {
+    if (!this.state.carts.length) {
+      return;
+    }
+
+    this.testTOTAL();
+  }
+
+  deleteItem = deletedItem => {
+    const remainItem = this.state.carts.filter(item => {
+      return item !== deletedItem;
+    });
+
+    this.setState(
+      {
+        carts: remainItem,
+      },
+      // this.getSubTotal(),
+    );
+
+    this.fetchDeleteItem(deletedItem);
+  };
+
+  fetchDeleteItem = deletedItem => {
+    fetch(`${SERVER}/order/cart/${deletedItem.productStockId}`, {
+      method: 'DELETE',
+      headers: {
+        Authorization: sessionStorage.getItem('access_token'),
+      },
+    })
+      .then(response => response.json())
+      .then(res => console.log(res));
+  };
+
+  goToCheckout = () => {
+    this.state.carts.length !== 0 &&
+      this.setState({ checkout: !this.state.checkout });
+
+    console.log('결제할거야');
+    fetch(`${SERVER}/order/checkout/${this.state.orderNumber}`, {
+      method: 'GET',
+      headers: {
+        Authorization: sessionStorage.getItem('access_token'),
+      },
+    })
+      .then(response => response.json())
+      .then(res => this.setUserInfo(res.data.user));
+    // .then(res => console.log(res.data.user));
+  };
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  setUserInfo = info => {
+    console.log(info);
+    this.setState({
+      name: info.userName,
+      email: info.email,
+      address: info.address,
+      zip: info.zipcode,
+      phoneNumber: info.phoneNumber,
+    });
+  };
+
+  placeOrder = props => {
+    console.log('주문버튼 누름');
+  };
+
+  render() {
+    const { carts, subtotal } = this.state;
+
+    return (
+      <div className="cart">
+        {carts.length === 0 && (
+          <div className="quiz">
+            <p>
+              Need some guidance? <span>Take the quiz</span>
+            </p>
+          </div>
+        )}
+        <section className={carts.length === 0 && 'onQuiz'}>
```

Comment:
boolean값을 이용해 조건을 작성해볼까요?
```suggestion
        <section className={!carts.length && 'onQuiz'}>
```